haystack/document_store/elasticsearch.py

```diff
@@ -872,7 +871,6 @@ def _convert_es_hit_to_document(
             text=hit["_source"].get(self.text_field),
             meta=meta_data,
             score=score,
-            probability=probability,
```

Comment:
We can simplify the if-block above as probability is not needed anymore. Probably we should apply the scaling part to the `score` now to be again in range 0-1

Reply:
Agreed. I will simplify the if-block and apply the scaling to the `score` variable.